apl/tabular-operators/lookup-operator.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: lookup
+description: 'This page explains how to use the lookup operator in APL.'
+---
+
+## Introduction
+
+The `lookup` operator extends a primary dataset with a lookup dataset based on a specified key column. It retrieves matching rows from the lookup dataset and appends relevant fields to the primary dataset. You can use `lookup` for enriching event data, adding contextual information, or correlating logs with reference tables.
```

Comment:
s/dataset/data table/

I talked about it in a comment further down, which is gonna be confusing :-)